src/test/cypress/global.d.ts

```diff
@@ -0,0 +1,42 @@
+/// <reference types="cypress" />
+interface TestHour {
+    calendarId: string;
+    startDate: string;
+    startTime: string;
+    endDay: string;
+    endTime: string;
+}
+
+interface TestException {
+    calendarId: string;
+    name: string;
+    startDate: string;
+    endDate: string;
+    openings: {
+        exceptionId: string,
+        startDate: string;
+        startTime: string;
+        endDay: string;
+        endTime: string;
+    }[]
+}
+
+interface TestCalendarReqBody{
```

Comment:
These types should already be specified for the application itself, can you reuse those?  They're in `src/types/types.ts`